src/PotentialEnergyEquationTerms.jl

```diff
@@ -5,11 +5,24 @@ using DocStringExtensions
 export PotentialEnergy
 
 using Oceananigans.AbstractOperations: KernelFunctionOperation
-using Oceananigans: Models.seawater_density
-using Oceananigans: Models.model_geopotential_height
+using Oceananigans.Models: seawater_density
+using Oceananigans.Models: model_geopotential_height
```

Comment:
I didn't realize that Oceananigans had a `model_geopotential_height` 😆 

Why isn't this just "geopotential_height"? What's different between the "height" and the "model height" ?

Reply:
I think it was because it gets the height from the model it is passed. It was added as a fallback for [`seawater_density`](https://github.com/CliMA/Oceananigans.jl/blob/fb9455bc10721a880d9911a39735e37f1c4961e0/src/Models/seawater_density.jl#L122-L125) here

https://github.com/CliMA/Oceananigans.jl/blob/fb9455bc10721a880d9911a39735e37f1c4961e0/src/Models/seawater_density.jl#L21-L24

I don't think there is any difference between "height" and "model height". The name could be changed to "geopotential_height".

Reply:
We can use `Oceananigans.geopotential_height` to avoid name conflicts

Reply:
Ah hmm we also have `model_temperature`. Perhaps we can leave it for now if this is mostly just a default...